### PR TITLE
Support for InfoBox

### PR DESCRIPTION
**Pull request by wuct**
#50

### CODE REVIEW

File: examples/gh-pages/scripts/components/basics/StyledMap.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react/addons";
+import {GoogleMaps} from "react-google-maps";
+import InfoBox from "react-google-maps/addons/InfoBox"
```
Comment (tomchentw) [inline examples/gh-pages/scripts/components/basics/StyledMap.js]: Missing `;`

File: src/addons/InfoBox.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+const {PropTypes} = React;
+
+class InfoBox extends SimpleChildComponent {
+  constructor (...args) {
+    super(...args);
+    this.state = {};
+  }
+
+  _createOrUpdateInstance () {
```
Comment (tomchentw) [inline src/addons/InfoBox.js]: Why not just use:
```js
_createOrUpdateInstance () {
  const GoogleMapsInfobox = require("google-maps-infobox");
  super._createOrUpdateInstance();
}
```

File: src/addons/InfoBox.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import SimpleChildComponent from "../internals/SimpleChildComponent";
+import createRegisterEvents from "../internals/createRegisterEvents";
+import exposeGetters from "../internals/exposeGetters";
+import InfoWindow from "../InfoWindow";
```
Comment (tomchentw) [inline src/addons/InfoBox.js]: Unused.

**Comment by tomchentw** [comment]
Please drop the commit `0187218` since we'll build it during release.

File: src/addons/InfoBox.js (hunk 2 above)
Comment (tomchentw) [inline src/addons/InfoBox.js]: Seems it doesn't assign `GoogleMapsInfoBox` under `google.maps`.

Comment (tomchentw) [inline src/addons/InfoBox.js]: Anyway, I'll do some refactor over `SimpleChildComponent#_createOrUpdateInstance` API and fix this issue. Please leave it as is, thanks!

Comment (tomchentw) [inline src/addons/InfoBox.js]: #55

File: examples/gh-pages/scripts/components/basics/StyledMap.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          closeBoxURL=""
+          position={myLatLng}
+          content={InfoBoxContent}/>
+      </GoogleMaps>
+    );
+  }
+
+}
+
+StyledMap.defaultProps = {
```
Comment (tomchentw) [inline examples/gh-pages/scripts/components/basics/StyledMap.js]: #53

**Comment by wuct** [comment]
Thanks @tomchentw!
I have removed a build file and fix an "unused" and a "missing `;`", please review.

**Comment by tomchentw** [comment]
squash:
16e7c00 >> 4e52c6c
176ab42 >> 613e5ef

reword:
`chore(examples):  build an example for InfoBox`

**Comment by wuct** [comment]
@tomchentw I just squashed and reworded them, please review. Thanks.

**Comment by tomchentw** [comment]
Awesome!

**Comment by tomchentw** [comment]
Released [v1.3.0](https://github.com/tomchentw/react-google-maps/releases/tag/v1.3.0). Thanks @wuct !

**Pull request merged**